spec/events/inventory_aggregate_spec.rb

```diff
@@ -381,10 +381,14 @@
     end
 
     it "should process a kit allocation event" do
-      kit = FactoryBot.create(:kit, :with_item, organization: organization)
+      # These are temp values used to prevent creating extra storage locations
```

Comment:
Yeah that's really bizarre. We should be able to specify something to the factory to avoid it doing this.

Reply:
Not sure I follow what you mean by specifying. You mean changing the factory to do something like:

```rb
factory :donation do
  storage_location { organization.storage_locations.first || create(:storage_location) }
end
```

or specify it in this line? `kit = FactoryBot.create(:kit, :with_item, organization: organization)`

The latter is kinda hard since the chain goes kit => line item => donation => storage location.

Reply:
I mean you should be able to make a trait or virtual attribute which we can key off of to have it not do this. We can pass the trait down to the child factories if necessary.